### PR TITLE
use html5 history to remove hash on urls

### PR DESCRIPTION
**Pull request by asmaNode**
Test are not failing on my env

### CODE REVIEW

File: env/openpaas.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    AUTH_PROVIDER_SETTINGS: {
+    authority: 'http://auth.open-paas.org.local/auth/realms/master',
+    client_id: 'openpaas',
+    redirect_uri: 'http://calendar.open-paas.org.local/calendar/auth/oidc/callback',
+    silent_redirect_uri: 'http://calendar.open-paas.org.local/calendar/auth/silent-renew.html',
+    post_logout_redirect_uri: 'http://calendar.open-paas.org.local/',
+    response_type: 'code',
+    scope: 'openid email profile'
+    }
+  };
```
Comment (tuanlc) [inline env/openpaas.js:16]: Why this change is included in this commit/PR?

Comment (asmaNode) [reply env/openpaas.js:16]: because we must add openpaas configuration for each SPA before building the image , it is not set by default

**Comment by alagane** [comment]
You are missing the `nginx.conf` file (see this pull request).
It is used on docker image, if you try a url different than `/calendar` (eg: landing on the page to edit user configurations) the url should not load without this file on docker.
See this PR: https://github.com/linagora/esn-frontend-inbox/pull/456/files

**Comment by asmaNode** [comment]
yes , i am pushing it just i am blocked with this error
```remote: Resolving deltas: 100% (6/6), completed with 6 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/main.
remote: error: At least 2 approving reviews are required by reviewers with write access.
To https://github.com/linagora/esn-frontend-calendar.git
 ! [remote rejected] #572 -> main (protected branch hook declined)
error: failed to push some refs to 'https://github.com/linagora/esn-frontend-calendar.git'

**Review by alagane — APPROVED** [review]

**Review by tanandy — APPROVED** [review]

**Comment by tanandy** [comment]
tests are failing

**Comment by asmaNode** [comment]
yes , i had explain on description , on my local i dont had this problem
